packages/cli/src/commands/MigrationCommandFactory.ts

```diff
@@ -80,7 +80,7 @@ export class MigrationCommandFactory {
       type: 'string',
       desc: 'Sets path to directory where to save entities',
     });
-    args.option('name', {
+    args.option('n', {
       alias: 'name',
```

Comment:
Here's an alternative fix if you don't want a short option.

```suggestion
    args.option('name', {
```